templates/example_mixing_score.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>This notebook will be polished enough that it doesn't need the "example" prefix, can copy the verb-based naming of the other notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>